templates/dhcpd.conf-extra.erb

```diff
@@ -2,4 +2,9 @@
 include "<%= @dhcp_dir %>/dhcpd.pools";
 include "<%= @dhcp_dir %>/dhcpd.hosts";
 include "<%= @dhcp_dir %>/dhcpd.ignoredsubnets";
+# extra_config
```

Comment:
Could you put this comment inside the conditional logic so it only shows up if there is extra config?